plugin/auth_ldap/tests/mtr/auth_ldap_simple.result

```diff
@@ -52,19 +52,19 @@ authentication_ldap_simple_group_search_filter	(|(&(objectClass=posixGroup)(memb
 authentication_ldap_simple_init_pool_size	10
 authentication_ldap_simple_log_status	1
 authentication_ldap_simple_max_pool_size	1000
-authentication_ldap_simple_server_host	172.18.0.2
+authentication_ldap_simple_server_host	127.0.0.1
 authentication_ldap_simple_server_port	389
 authentication_ldap_simple_ssl	OFF
 authentication_ldap_simple_tls	OFF
 authentication_ldap_simple_user_search_attr	uid
 SET GLOBAL authentication_ldap_simple_server_host = 'badhost';
-connect(localhost,zoidberg,zoidberg,test,13000,/work/build/mysql-test/var/tmp/mysqld.1.sock);
+connect(localhost,zoidberg,zoidberg,test,13000,/home/dutow/workspace/topics/ldap-fixup/80-local/mysql-test/var/tmp/mysqld.1.sock);
 ERROR 28000: Access denied for user 'zoidberg'@'localhost' (using password: YES)
-SET GLOBAL authentication_ldap_simple_server_host = '172.18.0.2';
+SET GLOBAL authentication_ldap_simple_server_host = '127.0.0.1';;
 SET GLOBAL authentication_ldap_simple_server_port = 388;
-connect(localhost,zoidberg,zoidberg,test,13000,/work/build/mysql-test/var/tmp/mysqld.1.sock);
+connect(localhost,zoidberg,zoidberg,test,13000,/home/dutow/workspace/topics/ldap-fixup/80-local/mysql-test/var/tmp/mysqld.1.sock);
 ERROR 28000: Access denied for user 'zoidberg'@'localhost' (using password: YES)
-SET GLOBAL authentication_ldap_simple_server_port = 389;
+SET GLOBAL authentication_ldap_simple_server_port = 389;;
```

Comment:
Probably `389` should be masked as well as it depends on `$MTR_LDAP_PORT` env variable.
Double semicolon.